modules/apps/adaptive-media/adaptive-media-image-impl/src/main/java/com/liferay/adaptive/media/image/internal/validator/AMImageValidatorImpl.java

```diff
@@ -75,19 +81,112 @@ public boolean isValid(FileVersion fileVersion) {
 			return false;
 		}
 
+		if (!_isFileVersionStoredMetadataSupported(fileVersion)) {
+			return false;
+		}
+
+		return true;
+	}
+
+	private boolean _checkDDMFormFieldsMapValue(
+		List<DDMFormFieldValue> ddmFormFieldValues,
+		long imageToolImageMaxValue) {
+
+		DDMFormFieldValue ddmFormFieldValue = ddmFormFieldValues.get(0);
```

Comment:
Can we name this method to something like `isLengthValid`? "check" should be used only when the method throws and exception on false values.

Reply:
👍